bot/seasons/valentines/movie_generator.py

```diff
@@ -0,0 +1,62 @@
+import logging
+import random
+from os import environ
+from urllib import parse
+
+import aiohttp
+import discord
+from discord.ext import commands
+
+log = logging.getLogger(__name__)
+
+
+class MovieGenerator:
+    """
+    a cog that gives random romance movie suggestion to a user
+
+    """
+
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.command()
+    async def romancemovie(self, ctx):
+        """
+
+        randomly selects romance movie and displays info about it
+        """
+
+        TMDB_API_KEY = environ.get("TMDB_API_KEY")
+        # selecting a random int to parse it to the page parameter
+        random_page = random.randint(0, 20)
+        # TMDB api params
+        params = {'api_key': TMDB_API_KEY,
+                  'language': 'en-US',
+                  'sort_by': 'popularity.desc',
+                  'include_adult': 'false',
+                  'include_video': 'false',
+                  'page': random_page,
+                  'with_genres': '10749'
+                  }
+        # the api request url
+        request_url = "https://api.themoviedb.org/3/discover/movie?" + parse.urlencode(params)
+        async with aiohttp.ClientSession() as session:
```

Comment:
This should probably be an attribute of the class instead of using a new client session every time, use the `__aenter__` and `__aexit__` (I think they are the function names) to create and clean up the client session

Reply:
I'm not sure what you mean by that, Although  i was told to check out scarymovie.py to get an example and they did it this way

Reply:
Oh in that case I guess it's fine, I just found it a little weird to create a new client session every command but I guess that it'd be cleaned up anyway.